sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.3.2" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.1" />
     <PackageReference Include="System.Collections.Immutable" Version="1.1.37" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />

```

Comment:
Is the `System.Runtime.InteropServices.RuntimeInformation` package still required? I assume not (it was the one that was added previously that cause the bug).

Reply:
Indeed it is no longer needed. I created a new PR for this: #3085.